orthoinference/src/main/java/org/reactome/orthoinference/StableIdentifierGenerator.java

```diff
@@ -32,8 +32,8 @@ public GKInstance generateOrthologousStableId(GKInstance inferredInst, GKInstanc
         // All Human PhysicalEntitys and Events will have a StableIdentifier instance in the stableIdentifier attribute
         GKInstance stableIdentifierInst = (GKInstance) originalInst.getAttributeValue(stableIdentifier);
         if (stableIdentifierInst == null) {
-            logger.fatal("No stable identifier instance found for " + originalInst + " -- terminating");
-            throw new RuntimeException("Could not find stable identifier instance");
+            logger.fatal("No stable identifier instance found for " + originalInst);
+            throw new RuntimeException("No stable identifier instance found for " + originalInst);
```

Comment:
Fantastic!  Now that the messages are in sync, you could extract it to a String variable and pass that to the log and exception.

If the message needs to be changed, it will keep them in sync with each other